mlops/internal_silos/YAML/setup_info.yml

```diff
@@ -0,0 +1,15 @@
+subscription_id: 6560575d-fa06-4e7d-95fb-f962e74efd7a
```

Comment:
You're right, however I'd rather adapt the example job to the setup created here, rather than the other way around. I've created an issue to capture this: https://github.com/Azure-Samples/azure-ml-federated-learning/issues/42